app/celery/scheduled_tasks.py

```diff
@@ -220,3 +221,46 @@ def check_templated_letter_state():
                 message=msg,
                 ticket_type=zendesk_client.TYPE_INCIDENT,
             )
+
+
+@notify_celery.task(name="heartbeart-inbox-sms")
+@statsd(namespace="tasks")
+def heartbeat_inbox_sms():
+    """
+    The function acts as a heartbeat to the inbox queue. Inbox is a list in Redis that
+    is made up of notifications. The post_api will push all the notifications to Inbox.
+    The heartbeat with check the inbox and move the notifications to the inflight list
+    accordingly. The heartbeat will then call the job that sends the list to the DB
+    and actually sends the sms.
+    """
+    # TODO: This will be instantiated from the app init.
+    sms_queue = MockQueue()
+
+    receipt_id_sms, list_of_sms_notifications = sms_queue.poll()
+
+    while list_of_sms_notifications:
+        process_inflight.apply_async((receipt_id_sms, list_of_sms_notifications, SMS_TYPE))
+        current_app.logger.info(f"SMS UUID: {receipt_id_sms} sent to Inflight List")
+        receipt_id_sms, list_of_sms_notifications = sms_queue.poll()
+
+
+@notify_celery.task(name="heartbeart-inbox-email")
+@statsd(namespace="tasks")
+def heartbeat_inbox_email():
+    """
+    The function acts as a heartbeat to the inbox queue. Inbox is a list in Redis that
+    is made up of notifications. The post_api will push all the notifications to Inbox.
+    The heartbeat with check the inbox and move the notifications to the inflight list
+    accordingly. The heartbeat will then call the job that sends the list to the DB
+    and actually sends the email.
```

Comment:
Same comment would apply here with the inbox, where we should refer to the queue.